vectorlm/utils/model_utils.py

```diff
@@ -25,6 +27,57 @@
 )
 
 
+def _is_bfloat_available() -> bool:
+    """
+    Return whether bfloat is supported for the
+    current CUDA device.
+
+    Returns:
+    --------
+        bool. True if bfloat is supported.
+    """
+    cuda_capability = torch.cuda.get_device_capability()
+    cuda_capability_str = "{}.{}".format(*cuda_capability)
+    if cuda_capability[0] >= 8.0:
+        print("Hardware capability {}; bfloat is supported".format(cuda_capability_str))
+        return True
+
+    else:
+        print(
+            "Hardware capability {}; bfloat isn't supported".format(cuda_capability_str)
+        )
+        return False
+
+
+def get_lora_model_from_base_model(
+    base_model: nn.Module, peft_config_dict: Dict
```

Comment:
dict annotation here also doesn't mention what are the types inside.

Reply:
Replaced with `dict[str, Any]` following the example of `fsdp_config`. Feel free to open a new refactor ticket if we want to make all config `dict` explicit (would indeed be quite helpful for programmers.)